forloop_modules/function_handlers/variable_handlers.py

```diff
@@ -1175,7 +1177,23 @@ def direct_execute(self, variable_name):
         """
     
     def input_execute(self, inp):
-        print(f'Variable value form PrintVariable icon: {inp("variable_name")}')
+        var_name = inp("variable_name")
+        var_obj = variable_handler.variables.get(var_name)
+
+        if var_obj is not None:
+            #message = f'PrintVariable: {var_name} = {var_obj.value}' ##Optional Variable printing format
+            message = f'{var_obj.value}'
+        else:
+            # fallback to printing the string itself
+            # message = f'PrintVariable: {var_name}' ##Optional Variable printing format
+            message = f'{var_name}'
+
+
+        # Send to FastAPI backend
+        try:
+            ncrb.post_console_log(message=message, type="print")
+        except Exception as e:
+            flog.warning(f"Failed to post console print log: {e}")
```

Comment:
I like that its not silent crash, but I think NCRB should care for the failed requests on its own - so it seems the whole try: except could be removed - you can try to test it

Reply:
? Can we get rid of the "try" "except" here? If the request crashes it should be handled in ncrb?